src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -243,37 +244,37 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		spawnproperties.setOpaque(false);
 
 		spawnproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/generate_overworld"),
-				new JLabel("Generate biome in overworld?")));
+				new JLabel(L10N.t("elementgui.biome.generate_overworld"))));
 		spawnproperties.add(spawnBiome);
 		spawnBiome.setSelected(true);
 		spawnBiome.setOpaque(false);
 
 		spawnproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/weight"),
-				new JLabel("<html>Biome weight:<br><small>Smaller is rarer biome")));
+				new JLabel(L10N.t("elementgui.biome.weight"))));
 		spawnproperties.add(biomeWeight);
 
 		spawnproperties.add(HelpUtils
-				.wrapWithHelpButton(this.withEntry("biome/type"), new JLabel("Biome type (temperature group):")));
+				.wrapWithHelpButton(this.withEntry("biome/type"), new JLabel(L10N.t("elementgui.biome.type"))));
 		spawnproperties.add(biomeType);
 
 		spawnproperties
-				.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/category"), new JLabel("Biome category:")));
+				.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/category"), new JLabel(L10N.t("elementgui.biome.category"))));
 		spawnproperties.add(biomeCategory);
 
-		spawnproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/parent"), new JLabel("Biome parent:")));
+		spawnproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/parent"), new JLabel(L10N.t("elementgui.biome.parent"))));
 		spawnproperties.add(parent);
 
 		spawnproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/dictionary"), new JLabel(
-				"<html>Biome dictionary types:<br><small>"
-						+ "Used by some mods to identify biome types for spawning")));
+				L10N.t("elementgui.biome.dictionnary")

```

Comment:
Don't split strings like this into separate entries. Note that this is valid for all similair cases.